s2n.mk

```diff
@@ -40,7 +40,7 @@ endif
 
 DEFAULT_CFLAGS += -pedantic -Wall -Werror -Wimplicit -Wunused -Wcomment -Wchar-subscripts -Wuninitialized \
                  -Wshadow  -Wcast-align -Wwrite-strings -fPIC -Wno-missing-braces\
-                 -D_POSIX_C_SOURCE=200809L -O2 -I$(LIBCRYPTO_ROOT)/include/ \
```

Comment:
Do we need to drop -O2 here? I think the later -O options will override whatever came before.

Reply:
that's right, we don't need to change this.